var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -33,10 +33,11 @@ class Brayns(CMakePackage):
     depends_on('deflect ~deflect-qt', when='+deflect')
     depends_on('freeimage')
     depends_on('glew', when='+viewer')
+    depends_on('libarchive')
     depends_on('libjpeg-turbo', when='+net')
     depends_on('libuv', when='+net')
     depends_on('opengl', when='+viewer')
-    depends_on('ospray', when='+ospray')
+    depends_on('ospray %intel', when='+ospray')
```

Comment:
Discussed briefly with @ppodhajski: adding `%intel` here makes this package buildable with Intel only. We should move this constraint outside package definition and do that as part of CI (e.g. `spack install brayns ^osrpray%intel`.

We would like to keep package recipes as generic as possible so that they can be build by different compilers and pushed upstream.

Reply:
Makes sense. Checking with @ppodhajski how to adapt the CI scripts accordingly then.